src/fibonacci.js

```diff
@@ -0,0 +1,9 @@
+export default function fibonacci(num) {
+  var sequence = [0,1];
+  for (var i = 1; i <= num -2; i++) {
```

Comment:
so this works for larger fibonacci sequences because you are starting with two numbers already in your sequence. However you want to structure your functions to be as versatile as possible by thinking of possible 'edge cases'. What if this was given 1, or 2 , or even 0.